src/js/ripple/remote.js

```diff
@@ -1038,6 +1038,47 @@ Remote.prototype.requestLedgerCurrent = function(callback) {
 };
 
 /**
+ * Request ledger_data
+ *
+ * Get the contents of a specified ledger
+ *
+ * @param {Object} options
+ * @property {Boolean} [options.binary]      - Flag which determines if rippled returns binary or parsed JSON
+ * @property {String} [options.ledger_hash]  - Hash of a ledger to get contents for
+ * @property {Number} [options.limit]        - Number of contents to retrieve from the ledger
+ * @property {Function} callback
+ *
+ * @callback
+ * @param {Error} error
+ * @param {LedgerData} ledgerData
+ *
+ * @return {Request} request
+ */
+
+Remote.prototype.requestLedgerData = function(options, callback) {
+  var request = new Request(this, 'ledger_data');
+
+  request.message.binary = options.binary !== false;
+  request.message.ledger_hash = options.ledger_hash;
```

Comment:
This command also accepts `ledger_index`. Probably best to reuse the request.selectLedger heuristic